src/components/filters/AttributeFilter/AttributeDropdown.tsx

```diff
@@ -128,16 +119,9 @@ export function loadAttributeElements(
     });
 }
 
-function getElementId(element: IAttributeElement) {
-    return element.uri.split("=")[1];
-}
-
 export function createAfmFilter(id: string, selection: IAttributeElement[], isInverted: boolean) {
-    return {
-        id,
-        type: "attribute",
-        [isInverted ? "notIn" : "in"]: selection.map(getElementId),
-    };
+    const attributeFilter = isInverted ? "negativeAttributeFilter" : "positiveAttributeFilter";
```

Comment:
Using Model["string key"] makes it easier to miss this during refactoring
```
const filterFactory = isInverted ? Model.negativeAttributeFilter : Model.positiveAttributeFilter;
```

Reply:
Thx, updated.